internal/services/setup_assets_for_network_service.go

```diff
@@ -18,13 +18,15 @@ import (
 type AssetsNetworkMapType map[utils.NetworkType][]data.Asset
 
 var StellarAssetsNetworkMap = AssetsNetworkMapType{
-	utils.PubnetNetworkType:  []data.Asset{assets.EURCAssetPubnet, assets.USDCAssetPubnet, assets.XLMAsset},
-	utils.TestnetNetworkType: []data.Asset{assets.EURCAssetTestnet, assets.USDCAssetTestnet, assets.XLMAsset},
+	utils.PubnetNetworkType:    []data.Asset{assets.EURCAssetPubnet, assets.USDCAssetPubnet, assets.XLMAsset},
+	utils.TestnetNetworkType:   []data.Asset{assets.EURCAssetTestnet, assets.USDCAssetTestnet, assets.XLMAsset},
+	utils.FuturenetNetworkType: []data.Asset{assets.XLMAsset},
```

Comment:
Why is USDC not available here ?

Reply:
I don't think Circle officially supports futurenet. [This page](https://www.circle.com/multi-chain-usdc/stellar) only lists pubnet and testnet:
<img width="367" alt="Screenshot 2024-12-02 at 9 21 38 AM" src="https://github.com/user-attachments/assets/736597b5-0552-462f-a5ea-e3237b0e405b">